package.json

```diff
@@ -27,19 +28,22 @@
     "winston"
   ],
   "dependencies": {
-    "node-loggly-bulk": "^2.0.1",
-    "clone": "^2.1.1",
-    "winston": "^3.0",
-    "winston-transport": "^4.2.0"
+    "node-loggly-bulk": "^2.2.4",
+    "clone": "^2.1.2",
+    "winston": "^3.2",
+    "winston-transport": "^4.3.0"
```

Comment:
These two should really really be in peer dependencies, we shouldn't force users to use our chosen version of winston... But moving it there can break stuff for existing users which will now have to add the winston dependency manually... Let's make a separate task for that.

Reply:
Agree.